src/lib/api.js

```diff
@@ -52,6 +49,8 @@ module.exports = function (opts, callback) {
                 opts.uri = parts[0] + '?' + qs;
 
             }
+        } else {
+            opts['X-Requested-With'] = 'XMLHttpRequest';
```

Comment:
whoops thanks!

As for doing that at the top – but then wouldn't we have to override it every time inside `if (opts.hots !== '')`? That's what I was trying to avoid.